dive_core/common/emulate_pm4.h

```diff
@@ -246,6 +248,7 @@ class EmulatePM4
         IbLevel  m_top_of_stack;
         IbStack *GetCurIb() { return &m_ib_stack[m_top_of_stack]; }
     };
+    static_assert(kMaxNumIbLevel == EmulateState::kTotalIbLevels);
```

Comment:
why not just use either one of them if they have the same value?

Reply:
Oh you're right! I was originally hesitant to expose the enum outside, but it makes sense to in this case. Much better than having the same value duplicated!